src/Serilog.AspNetCore/AspNetCore/RequestLoggingMiddleware.cs

```diff
@@ -29,15 +30,17 @@ class RequestLoggingMiddleware
         readonly RequestDelegate _next;
         readonly DiagnosticContext _diagnosticContext;
         readonly MessageTemplate _messageTemplate;
+        readonly Func<HttpStatusCode, LogEventLevel> _getLogEventLevel;
```

Comment:
The level might be determined from more than just the status code, would accepting `HttpContext` be more appropriate, here?

Reply:
I was trying to mimic the current behavior as close as possible  :). Sure no problem, I will pass the `HttpContext` to the `Func`.